packages/contentstack-utilities/src/flag-deprecation-check.ts

```diff
@@ -8,9 +8,14 @@ import cliux from './cli-ux';
  */
 export default function (deprecatedFlags = [], suggestions = [], customMessage?: string) {
   return (input, command) => {
+    const { context: { flagWarningPrintState = {} } = {} } = command
     let isCommandHasDeprecationFlag = false;
     deprecatedFlags.forEach((item) => {
       if (command.argv.indexOf(item) !== -1) {
+        if (flagWarningPrintState[command.id + item]) {
```

Comment:
Please verify this logic, when you multiple deprecation flags

Reply:
@shafeeqd959 verified for the mentioned scenario